### PR TITLE
Html changes

### PR DESCRIPTION
**Pull request by muydanny**
Added random photos that appear if a review doesnt have one and made the new shelter form look slightly crispier

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -194,6 +204,23 @@
 .bold {
   font-weight: bold;
 }
+
+.bolder{
+  font-weight: bold;
+  text-align: center;
+  width: 400px;
+  height: 30px;
+  border-radius: 10px;
```
Comment (muydanny) [inline app/assets/stylesheets/application.css:213]: made the boxes round. next level stuff right here

File: app/controllers/apps_controller.rb [hunk 2]
```diff
@@ -18,6 +18,9 @@ def show
 
   def create
     @app = App.create(app_params)
+    if @app.image == nil
```
Comment (muydanny) [inline app/controllers/apps_controller.rb]: used those hand dandy params and realized nil is not what im looking for

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -20,6 +20,11 @@ def update
   def create
     @review = Review.create(shelter_id: shelter_id[:id])
     @review.update(review_params)
+
+    if @review.image == ""
```
Comment (muydanny) [inline app/controllers/reviews_controller.rb:24]: but instead an empty string

Comment (perryr16) [reply app/controllers/reviews_controller.rb:24]: This looks great

**Pull request merged**